doc/src/manual/variables-and-scoping.md

```diff
@@ -152,27 +150,18 @@ julia> z
 10
 ```
 
-Multiple global or local definitions can be on one line and can also be paired with assignments:
```

Comment:
The extra newline should probably be fixed... syntax deprecations get thrown from inside the parser I think. In 1.0 I guess this syntax would be an error, in which case hopefully the error message will indicate the correct syntax in a clear way?